draft-irtf-cfrg-opaque.md

```diff
@@ -339,8 +339,8 @@ OPAKE was taken.
 OPAQUE relies on the following cryptographic protocols and primitives:
 
 - Oblivious Pseudorandom Function (OPRF, {{!I-D.irtf-cfrg-voprf}}, version -06):
-  - Blind(x): Convert input `x` into an element of the OPRF group, randomize it
-    by some scalar `r`, producing `M`, and output (`r`, `M`).
+  - Blind(x): Convert input `x` into an element of the OPRF group, multiply it
+    with some random scalar `r`, producing `M`, and output (`r`, `M`).
```

Comment:
We wrote this intentionally as randomize as that does not assume additive or multiplicative blinding -- can we please revert?

Reply:
Done ;)